.github/workflows/mypy_check.yml

```diff
@@ -0,0 +1,15 @@
+name: "mypy check"
+on: [pull_request]
+
+jobs:
+
+  static-type-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v3
+      with:
+        python-version: '3.x'
```

Comment:
Can we pick a specific version of Python (say 3.8 to match oppia-web)? We like pinning all our versions so that an upgrade doesn't surprise us and break something

Reply:
I have specified the version, 3.8.